src/components/Badge.tsx

```diff
@@ -37,5 +38,5 @@ export default withTheme(
 				{text}
 			</BaseBadge>
 		);
-	},
-);
+	}),
+) as React.ComponentClass<BadgeProps>;
```

Comment:
Why is this cast required?

Reply:
To keep me sane! I think we just started using this pattern to get consistent output between `compose` and `withTheme`. The styled components typings are not great..